docs/en/observability/synthetics-run-test.asciidoc

```diff
@@ -18,12 +18,14 @@ application code, <<synthetics-test-suite,use a test suite>>.
 A test suite can contain any number of journeys in any number of files using the file extensions `.journey.ts` or `.journey.js`.
```

Comment:
That's already the terminology we use with push. There's even a `--project` flag we use when pushing monitors to scope them to a particular project prefix.

CC @vigneshshanmugam @hbharding may have some input on this too.

I do think it's worth starting to reference suites as projects sooner rather than later so that the users we onboard start with the right terminology in their heads.

Might be worth adding a callout to say we've recently changing the name. Although I don't think we actually need to. Just something to consider.